rules/Proj0800.md

```diff
@@ -0,0 +1,52 @@
+# Proj0800: Configure Central Package Version Management explicitly
+Dependency management is a core feature of NuGet. Managing dependencies for a
+single project can be easy. Managing dependencies for multi-project solutions
+can prove to be difficult as they start to scale in size and complexity. In
+situations where you manage common dependencies for many different projects,
+you can leverage NuGet's central package management (CPM) features to do all of
+this from the ease of a single location.
+
+Hence, this rules advises to enable Central Package Version Management via
+enabling it with `<DevelopmentDependency>true</DevelopmentDependency>`.
```

Comment:
After reading https://learn.microsoft.com/en-us/nuget/consume-packages/central-package-management, https://github.com/NuGet/Home/wiki/DevelopmentDependency-support-for-PackageReference and https://github.com/NuGet/Home/issues/4125 I don't how the 2 concepts relate to each other.
I do see a suggestion for Proj1101 Package references should have stable versions, which shouldn't be triggered for DevelopmentDependencies.

Reply:
I think that what you refer to might be an interesting new issue and/or rule. But it should have been `<ManagePackageVersionsCentrally>`. Has been fixed.